MathlibTest/CategoryTheory/ConcreteCategory/AlgebraCat.lean

```diff
@@ -0,0 +1,48 @@
+import Mathlib
+
+universe v u
+
+open CategoryTheory AlgebraCat
+
+set_option maxHeartbeats 10000
```

Comment:
Maybe synthesis heartbeats also. As to the precise bounds, I'm not sure.

Reply:
Done.

Reply:
Great!